phpcs.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
+	<description>Generally-applicable sniffs for WordPress plugins</description>
+
+	<rule ref="WordPress-Extra" />
+	<rule ref="WordPress-Docs" />
+	<rule ref="WordPress-VIP-Go" />
+
+	<!--
```

Comment:
If it ain't needed, it shouldn't be there ;)

Reply:
Agreed. https://github.com/Automattic/newspack-ads/pull/3/commits/4eda0770591dcc5aca9132ff6a887243bd035a76